ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -1409,9 +1409,9 @@ class SignalProducerSpec: QuickSpec {
 			}
 
 			describe("disposal") {
-				var completeOuter: (Void -> Void)!
-				var disposeOuter: (Void -> Void)!
-				var execute: (FlattenStrategy -> Void)!
+				var completeOuter: (() -> ())!
+				var disposeOuter: (() -> ())!
+				var execute: (FlattenStrategy -> ())!
```

Comment:
Let's just be consistent in this PR. I don't have a strong opinion and would be open to changing to `Void`. (Particularly if we're going to match the Swift stdlib style.) But let's do that consistently across the codebase if we're going to do it.